scripts/run.sh

```diff
@@ -264,21 +277,10 @@ run_simulator() {
   --priority-fee=1
 }
 
-if [[ ${SKIP_NETWORK_RUNNER_START} == false ]]; then
-  echo "running ginkgo"
-  run_ginkgo
-  # to fail the script if ginkgo failed
-  EXIT_CODE=$?
-else
-  echo "running scripts/parser/main.go"
-  go run scripts/parser/main.go \
```

Comment:
This is really hacky... we should deprecate this and just unify to `tests/e2e/e2e_test.go`...